### PR TITLE
Remove Python 2 install instructions from docs/README

### PR DESCRIPTION
**Pull request by CyberDem0n**
docs/README.rst mainly duplicates README.rst and also should be changed. Besides that remove test/coverage badges.

followup on #2821

### CODE REVIEW

File: docs/README.rst [hunk 1]
```diff
@@ -186,3 +181,8 @@ That said, here are some pieces of your infrastructure you should be sure to tes
 * ``kill -9`` of any postgres process (except postmaster!). This is a decent simulation of a segfault.
 
 One thing that you should not do is run ``kill -9`` on a postmaster process. This is because doing so does not mimic any real life scenario. If you are concerned your infrastructure is insecure and an attacker could run ``kill -9``, no amount of HA process is going to fix that. The attacker will simply kill the process again, or cause chaos in another way.
+
+.. |Tests Status| image:: https://github.com/zalando/patroni/actions/workflows/tests.yaml/badge.svg
```
Comment (hughcapet) [inline docs/README.rst:185]: do we need it here at all?

Comment (CyberDem0n) [reply docs/README.rst:185]: Good question. It doesn't look that readthedocs even renders it.

File: docs/README.rst [hunk 2]
```diff
@@ -186,3 +181,8 @@ That said, here are some pieces of your infrastructure you should be sure to tes
...
+   :target: https://github.com/zalando/patroni/actions/workflows/tests.yaml?query=branch%3Amaster
+.. |Coverage Status| image:: https://coveralls.io/repos/zalando/patroni/badge.svg?branch=master
+   :target: https://coveralls.io/github/zalando/patroni?branch=master
```
Comment (CyberDem0n) [inline docs/README.rst:188]:
```suggestion
```

File: docs/README.rst (hunk 1 above)
Comment (hughcapet) [reply docs/README.rst]: exactly

**Comment by hughcapet** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**